tests/integration/helpers/sort-by-test.js

```diff
@@ -301,21 +301,38 @@ module('Integration | Helper | {{sort-by}}', function(hooks) {
     assert.equal(find('*').textContent.trim(), 'abc', 'cab is sorted to abc');
   });
 
-  test('ignores undefined values sorting', async function (assert) {
+  test('it sorts undefined values last', async function(assert) {
     this.set('array', [
-      { name: 'c' },
-      { name: 'a' },
-      { name: undefined },
```

Comment:
One behaviour of sort was to remove undefined values.  We would probably want to keep this sort of test around.

Reply:
Sorry, not sure I understand -- what do you mean by "remove undefined values"? 

This merely changes the test to assert that the undefined value is at the end of the resulting array. At present the test will assert `"abc"` regardless of what position `undefined` is in the resulting array since undefined renders nothing, so `textContent` will be `abc` no matter what

You could change sort-by to sort undefined first, sort undefined last, remove undefineds or even randomly add a bunch of undefineds, and this test will always pass before this change, so it doesn't really test anything at all

Reply:
See original assertion...

```
assert.equal(find('*').textContent.trim(), 'abc');
```

Reply:
Sorry I've seen that and I still don't understand. I've been relooking the test over and over again, could you please elaborate in more detail?

If `sort-by` returns any of these results, `textContent` is still `'abc'`:
```hbs
{{#each (array "a" "b" "c") as |str|}}{{str}}{{/each}}
textContent === 'abc'

{{#each (array "a" "b" undefined "c") as |str|}}{{str}}{{/each}}
textContent === 'abc'

{{#each (array undefined "a" undefined undefined "b" undefined "c" undefined) as |str|}}{{str}}{{/each}}
textContent === 'abc'
```

I'm not sure what the original assertion is testing for?

Put another way: what result of `sort-by` would cause this assertion to fail?

Reply:
We simply want the original test around.  The test assertions you added should be in a new test